docs/fastdds/use_cases/tcp/tcp_with_multicast_discovery.rst

```diff
@@ -30,6 +31,14 @@ while at the same time have the flexibility of automatic discovery.
           :end-before: <!--><-->
           :lines: 2-4, 6-13, 15-16
 
+   .. tab:: C++
+
+      .. literalinclude:: ../../../../code/DDSCodeTester.cpp
+        :language: c++
+        :dedent: 8
+        :start-after: //LARGE_DATA_BUILTIN_TRANSPORTS
+        :end-before: //!
+
 .. note::
```

Comment:
Adding the comment here as I cannot add it in the code-block below. 

I'd add the code-block in the note. If this is not possible add another section with the note and the code just for stating how to disable SHM